webapp/src/pages/EditChatroomMemberPage/components/ToggleSwitchSet.tsx

```diff
@@ -78,6 +94,15 @@ export default function ToggleSwitchSet({
           updateChatroomMemberDto: dto,
         });
         setUpdateChatroomMemberDto({ ...oldUpdateChatroomMember, ...dto });
+        const { username } = canEditParams.destUser!;
+        try {
+          const key = dtoToKey(dto);
+          notify(
+            `${username} chatroom member successfully set to ${key} = ${dto[key]}`,
```

Comment:
```suggestion
            dto[key]
              ? `Added ${key} role to ${username}`
              : `Removed ${key} role from ${username}`,
```

suggestion: What do you think about this? To make it more user friendly.

Reply:
Done